moto/rds/models.py

```diff
@@ -1651,24 +1662,17 @@ def __init__(self, region_name: str, account_id: str):
         self.db_proxies: Dict[str, DBProxy] = OrderedDict()
 
     def reset(self) -> None:
-        self.neptune.reset()
         super().reset()
 
-    @property
-    def neptune(self) -> NeptuneBackend:
-        return neptune_backends[self.account_id][self.region_name]
+    def db_cluster_options(self, engine) -> List[Dict[str, Any]]:  # type: ignore
+        from moto.rds.utils import decode_orderable_db_instance
 
-    @property
-    def db_cluster_options(self) -> List[Dict[str, Any]]:  # type: ignore
-        if self._db_cluster_options is None:
```

Comment:
Can we revert this part? I saw the commit where you had to revert the `@cache`-property, but that does mean we'd need our manual cache back - this is a very expensive operation, IIRC

Reply:
Done.  The `cache` decorator is not available in 3.8, but `lru_cache` is, and they are functionally equivalent for our purposes here.